libraries/habitat_shared.rb

```diff
@@ -26,8 +26,6 @@ def hab(*command)
 
       if Gem::Requirement.new('>= 14.3.20').satisfied_by?(Gem::Version.new(Chef::VERSION))
```

Comment:
If you're taking out the else here you should just take this whole thing out and require Chef 14.3. If you want to support 14.3 then put a cookstyle comment on that line to avoid the alert

Reply:
okay, I wasn't sure, currently the cookbook supports back to client 12.20 and I didn't want to break any of it. would any of these changes effect that?

Reply:
You'll want to turn that cop off with a comment then.

Reply:
ok has that syntax changed from the foodcritic method?

Reply:
# cookstyle: disable x/y

Reply:
great thank you!